src/main/java/org/isf/dicom/manager/DicomManagerFactory.java

```diff
@@ -48,7 +48,7 @@ public static Long getMaxDicomSizeLong() throws OHDicomException {
 	 * return the manager for DICOM acquired files
 	 * @throws OHDicomException 
 	 */
-	public synchronized static DicomManagerInterface getManager() throws OHDicomException {
+    public static synchronized DicomManagerInterface getManager() throws OHDicomException {
```

Comment:
could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

Reply:
As noted in the other PR I am using it but it appears that either the files have some mix of indentation when I started or the default tab width is not set or defined.